src/test/kotlin/org/partiql/scribe/targets/partiql/PartiQLTargetSuite.kt

```diff
@@ -1,23 +1,20 @@
 package org.partiql.scribe.targets.partiql
 
-import org.partiql.plugins.local.LocalConnector
 import org.partiql.scribe.targets.SqlTargetSuite
-import org.partiql.scribe.test.SessionProvider
+import org.partiql.scribe.utils.Functions
+import org.partiql.scribe.utils.SessionProvider
 import kotlin.io.path.toPath
 
 class PartiQLTargetSuite : SqlTargetSuite() {
-
     override val target = PartiQLTarget
 
     override val root = this::class.java.getResource("/outputs/partiql")!!.toURI().toPath()
 
-    override val sessions = SessionProvider(
-        mapOf(
-            "default" to
-                    LocalConnector.Metadata(
-                        this::class.java.getResource("/catalogs/default")!!.toURI().toPath(),
-                        listOf(split)
-                    )
+    override val sessions =
+        SessionProvider(
+            scalarOverloads =
+                mapOf(
+                    "split" to Functions.scalarSplit,
+                ),
         )
```

Comment:
It seems that `split` is added as an overload to all target test suites in order to test that UDFs can be included and transpiled. Perhaps we should comment to that effect.

Reply:
Sure, I'll add a comment